app/views/groups/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="d-flex justify-content-between">
+    <h3>All Categories</h3>
+    <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
+  </div>
+
+  <div class="categories">
+    <% @groups.each_with_index do |cat, index|%>
+      <%= link_to group_path(cat) do %>
+        <div class="category card mb-3">
+          <div class="row g-0">
+             <div class="col-md-4">
+             <%= image_tag(cat.icon, class: "img-fluid", alt: cat.name, style: "height: 100px; width:100px;") if cat.icon.attached? %>
+        </div>
+            <div class="col-md-8">
+              <div class="card-body text-dark">
+                <div class="d-flex justify-content-between">
```

Comment:
- [ ] Great job 👏 , however, let's stick to the design flow for the category as it is on the transaction which was a pattern provided.
- The top bar should have a blue background. The color used is `#3778c2`
- the "Add new category" should be a button at the bottom of the page. The color used is `#5fb523`

### Expected design:
![image](https://github.com/arnoldnekemiah/Budget-app/assets/50990925/8e069c74-ceff-4654-aa36-288a8b77b903)

### Your design:
![image](https://github.com/arnoldnekemiah/Budget-app/assets/50990925/589be698-1651-4086-b347-dbd1e819f063)